### PR TITLE
Version 0.9.5

### PR DESCRIPTION
**Pull request by ihhub**
another step forward

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,62 @@
...
+- AI heroes do not block way for other heroes anymore
+- use spacebar only for activating object
+- fix missing UI update in hero meeting dialog while moving artifacts within internal hero's dialog
+- speed up campaign game menu opening time
+- fix incorrect object rendering order on World Map
+- after loading a map and starting a new game the same map will be selected in the list of maps
+- improve AI logic to capture defendless castles
+- fix black screen for loading a campaign map without files
+- allow AI hero to revisit a castle more than once
+- fix a case of splitting mosnter stack for 2 monsters in stack
```
Comment (felix642) [inline changelog.txt:22]: Don't know if it matters, but there's a small typo here. (mosnter)

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,62 @@
...
+- add new UI text rendering classes
+- fix issue when abandoned mine becomes OBJ_ZERO when captured by AI-controlled hero
+- fix missing road identification on broken by editor maps
+- make per-user config file as default option
+- allow attacking another hero located on a Temple
+- fix attacking monster sprite behavior
+- fix number of choices when splitting a troop into multiple troops
+- fix incorrect trading rate during battle
+- fix missing shadow removal of objects under a boat
+- show Dwarf's resitance message in the log only after spell completion
```
Comment (felix642) [inline changelog.txt:54]: Another typo here (resitance)

**Review by felix642 — COMMENTED** [review]
Hi @ihhub I saw 2 small typos if you want to fix them before merging.

**Comment by ihhub** [comment]
> Hi @ihhub I saw 2 small typos if you want to fix them before merging.

Thanks, @felix642 ! I'll fix them.

**Pull request merged**